tpl/departures.js

```diff
@@ -18,11 +19,16 @@ const time = (t) => h.time({
 	moment(t).tz(timezone).locale(locale).format('LT')
 ])
 
+const delay = (d) => d > 1000 ? ` +${ms(d)}` : ''
```

Comment:
The delay can also be negative. AFAIR we need to cater for that since `ms` doesn't work properly with negative numbers.